src/utils/prim-lexing.fs

```diff
@@ -201,7 +201,7 @@ namespace Internal.Utilities.Text.Lexing
         let numUnicodeCategories = 30 
         let numLowUnicodeChars = 128 
         let numSpecificUnicodeChars = (trans.[0].Length - 1 - numLowUnicodeChars - numUnicodeCategories)/2
-        let lookupUnicodeCharacters (state,inp) = 
+        let lookupUnicodeCharacters state inp =
```

Comment:
This change is harmless but AFAICS doesn't alter the representation or calls of the function?  e.g. for 
```
type C() = 
   let f (x,y) = x + y
   member a.M(b,c) = f (b,c)
```
we get

```
.method assembly hidebysig instance int32 
        f(int32 x,
          int32 y) cil managed

```